ta/arch/arm/link.mk

```diff
@@ -49,7 +49,7 @@ cleanfiles += $(link-out-dir$(sm))/dyn_list
 
 link-ldadd  = $(user-ta-ldadd) $(addprefix -L,$(libdirs))
 link-ldadd += --start-group $(addprefix -l,$(libnames)) --end-group
-ldargs-$(user-ta-uuid).elf := $(link-ldflags) $(objs) $(link-ldadd)
+ldargs-$(user-ta-uuid).elf := $(link-ldflags) $(objs) $(link-ldadd) $(libgcc$(sm))
```

Comment:
I think `$(libgcc$(sm))` is needed in `link_shlib.mk` too: https://github.com/OP-TEE/optee_os/blob/3.8.0/ta/arch/arm/link_shlib.mk#L27